docs/concepts/what-is-a-schedule-to-close-timeout.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 A Schedule-To-Close Timeout is the maximum amount of time allowed for the overall [Activity Execution](/concepts/what-is-an-activity-execution), from when the first [Activity Task](/concepts/what-is-an-activity-task) is scheduled to when the last Activity Task, in the chain of Activity Tasks that make up the Activity Execution, reaches a Closed status.
 
-- [How to set a Schedule-To-Close Timeout in Go](/go/how-to-set-a-schedule-to-close-timeout-in-go)
+- [How to set a Schedule-To-Close Timeout](/go/how-to-set-a-schedule-to-close-timeout-in-go)
```

Comment:
Why was "in Go" removed from the Go-specific links on the non-Go-specific page?

Reply:
The Go specific page link has a redirect so it connects to the App Dev Guide, and opens on the user's preferred SDK tab. Leaving it in there for now because it works. Will tackle all the links updates next in a different PR.